app/views/shared/proxy_rules/_form.html.slim

```diff
@@ -1,7 +1,7 @@
 = form.inputs do
   = form.input :http_method, as: :select, collection: ProxyRule::ALLOWED_HTTP_METHODS, include_blank: false, label: 'Verb'
   = form.input :pattern
-  = form.input :redirect_url, as: :string if proxy_rule.owner.try(:using_proxy_pro?)
+  = form.input :redirect_url, as: :string if local_assigns[:include_redirect]
```

Comment:
Good enough for now.

One way to avoid that is to have a presenter ProxyRuleOwnerPresenter
So it can be a BackendApi or a Service and logic is held by the ptresenter without using `local_assigns` 

Especially you might forget this assignments in many of your render partials and it works but hits a bug
Also it will be DRY